src/Vlingo.Actors.Tests/WorldTest.cs

```diff
@@ -76,6 +76,36 @@ public void TestThatARegisteredDependencyCanBeResolved()
             Assert.Same(dep, result);
         }
 
+        [Fact]
+        public void TestThatResolvesAMissingDependencyReturnsNull()
+        {
+            var name = Guid.NewGuid().ToString();
+            var result = World.ResolveDynamic<IAnyDependecy>(name);
+            Assert.Null(result);
+        }
+
+        [Fact]
+        public void TestThatResolvesWrongByRefTypeReturnsNull()
+        {
+            var name = Guid.NewGuid().ToString();
+            var dep = 5;
+            World.RegisterDynamic(name, dep);
+
+            var result = World.ResolveDynamic<string>(name);
+            Assert.Null(result);
+        }
+
+        [Fact]
+        public void TestThatResolvesWrongByValTypeReturnsDefault()
+        {
+            var name = Guid.NewGuid().ToString();
+            var dep = "teststring";
+            World.RegisterDynamic(name, dep);
+
+            var result = World.ResolveDynamic<int>(name);
+            Assert.Equal(0, result);
```

Comment:
This would be expected for value objects, right?

Reply:
If you cast to wrong type the exception is normal. The goal was not to throw if absent in dictionary

Reply:
I would only include test that of not in dictionary it doesn’t throw. For wrong casting it will throw